kong.config.sh

```diff
@@ -30,12 +30,33 @@ PAYLOAD
     -d @- ) <<PAYLOAD
 {
     "name": "data-broker",
-    "uris": [/device/(.*)/latest", "/subscription"],
+    "uris": ["/device/(.*)/latest", "/subscription"],
     "strip_uri": false,
     "upstream_url": "http://data-broker:80"
 }
 PAYLOAD
 authConfig "data-broker"
+(curl -o /dev/null ${kong}/apis -sS -X POST \
+    --header "Content-Type: application/json" \
+    -d @- ) <<PAYLOAD
+{
+    "name": "data-streams",
+    "uris": ["/stream"],
```

Comment:
This endpoint is treated directly by socket.io library, right? (from data-broker)

Reply:
Quite the opposite - this is the endpoint that is handled by data-broker (issues one-time access tokens)